tests/robot/OBSERVATION/01_create.robot

```diff
@@ -98,6 +98,19 @@ Force Tags              observation_create    create
     observation.validate response - 201
 
 
+005.1 Create Sofa Score
+	[Documentation]     1. *CREATE* new EHR record\n\n
+	...                 2. *LOAD* _create-sofa-score.json_\n\n
+	...                 3. *UPDATE* ``Subject - Identifier - value`` with the _UUID:_ ${subject_id} which was created in EHR record\n\n
+    ...                 4. *POST* example JSON to observation endpoint\n\n
+	...                 5. *VALIDATE* the response status
+	[Tags]             	sofa-score    valid
+
+	ehr.create new ehr    000_ehr_status.json
+	observation.create sofa score  Sofa Score  create-sofa-score.json
+    observation.validate response - 201
+
+
```

Comment:
Same as above, please dont edit the tests. 
Just inform us (@wlad @af-mle @af-pwo ) if you find something suspicious and we have a look into it.
This 005.1 test can be deleted, we only do one happy path test per profile.

Reply:
@wlad mentioned the test structure might be refactored, there is no sense on separating tests for the same profile on different robot files.